compose.yaml

```
@@ -0,0 +1,39 @@
+services:
+  server:
+    build:
+      context: .
+      target: final
+    ports:
+      - 8080:8080
+    volumes:
+      - uploads-data:/app/wwwroot/uploads
+volumes:
+  db-data:  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)